# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/smartmontools/hotfix-samsung-t3.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- smartmontools-7.3/drivedb.h.orig	2022-11-20 16:22:56.534554848 +0100
+++ smartmontools-7.3/drivedb.h	2022-11-20 16:26:20.670563339 +0100
@@ -5235,6 +5235,12 @@
     "",
     "-d sat"
   },
+  { "USB: Portable SSD T3; ",
+    "0x04e8:0x61f3",
+    "", // 0x0100
+    "",
+    "-d sat"
+  },
   { "USB: Samsung Portable SSD T5; ",
     "0x04e8:0x61f5",
     "", // 0x0100
